exp-player/addon/components/exp-player.js

```diff
@@ -181,6 +183,10 @@ able to continue the study.
         },
         closeExitWarning() {
             this.set('hasAttemptedExit', false);
+        },
+        updateFramePage(framePage) {
+            this.set('framePage', framePage);
+            this.sendAction('updateFramePage', framePage);
```

Comment:
Since `framePage` is passed in just like `frameIndex`and should update when the component's value changes, can I remove sending the action?